docs/howto/add_relationship_links.md

```diff
@@ -15,40 +15,42 @@ class Api::V1::UsersController < ApplicationController
       serializer: Api::V1::UserSerializer,
       include: []
   end
+end
 ```
 
 Bear in mind though that ActiveModelSerializers are [framework-agnostic](outside_controller_use.md), Rails is just a common example here.
 
 ### Links as an attribute of a resource
-**This is applicable to JSONAPI, JSON and Attributes adapters**
+**This is applicable to JSON and Attributes adapters**
 
 You can define an attribute in the resource, named `links`.
 
 ```ruby
 class Api::V1::UserSerializer < ActiveModel::Serializer
-  attributes :id, :name, :links
+  include Rails.application.routes.url_helpers
```

Comment:
I suppose they should do it like [ActiveModelSerializers::Adapter::JsonApi::Link](https://github.com/rails-api/active_model_serializers/blob/0-10-stable/lib/active_model_serializers/adapter/json_api/link.rb) and include [ActiveModelSerializers::SerializationContext::UrlHelpers](https://github.com/rails-api/active_model_serializers/blob/0-10-stable/lib/active_model_serializers/serialization_context.rb#L14-L22)

```ruby
        include ActiveModelSerializers::SerializationContext::UrlHelpers
```